code/datums/components/squeek.dm

```diff
@@ -24,22 +24,21 @@
 	if(use_delay_override)
 		use_delay = use_delay_override
 
-	if(istype(parent, /atom))
-		RegisterSignal(COMSIG_ATOM_BLOB_ACT, .proc/play_squeak)
-		RegisterSignal(COMSIG_ATOM_HULK_ATTACK, .proc/play_squeak)
```

Comment:
refactor request, let me pass a list in of COMSIGs so we dont have to copypaste this shit